ftw_tools/torchgeo/datamodules.py

```diff
@@ -74,74 +91,104 @@ def __init__(
         print(f"Test countries: {self.test_countries}")
         print(f"Number of samples: {self.num_samples}")
 
-        self.train_aug = AugmentationSequential(
+        augs = [
             K.Normalize(mean=self.mean, std=self.std),
             K.RandomRotation(p=0.5, degrees=90),
             K.RandomHorizontalFlip(p=0.5),
             K.RandomVerticalFlip(p=0.5),
             K.RandomSharpness(p=0.5),
-            data_keys=["image", "mask"],
-        )
-        self.aug = AugmentationSequential(
-            K.Normalize(mean=self.mean, std=self.std), data_keys=["image", "mask"]
-        )
-        super().__init__(FTW, batch_size, num_workers, **kwargs)
+        ]
+        if random_shuffle:
+            print("Using random channel shuffle augmentation")
+            augs.append(kornia.contrib.Lambda(randomChannelShuffle))
+
+        if resize_factor is not None:
+            if resize_factor < 1:
+                raise ValueError("Resize factor must be >= 1")
+            print(f"Using resize factor of {resize_factor}")
+            augs.append(
+                K.Resize(
+                    (int(256 * resize_factor), int(256 * resize_factor)),
+                    resample=kornia.constants.Resample.BILINEAR,
```

Comment:
Should also set antialias=True here which is default to False in kornia's Resize for some reason

Reply:
Great, thanks!!